options.go

```diff
@@ -0,0 +1,67 @@
+package allure
+
+import (
+	"github.com/dailymotion/allure-go/severity"
+)
+
+type Option func(r hasOptions)
+
+func Lead(lead string) Option {
+	return func(r hasOptions) {
+		r.addLabel("lead", lead)
+	}
+}
+
+func Owner(owner string) Option {
+	return func(r hasOptions) {
+		r.addLabel("owner", owner)
+	}
+}
+
+func Epic(epic string) Option {
+	return func(r hasOptions) {
+		r.addLabel("epic", epic)
+	}
+}
+
+func Severity(severity severity.Severity) Option {
+	return func(r hasOptions) {
+		r.addLabel("severity", string(severity))
+	}
+}
+
+func Story(story string) Option {
+	return func(r hasOptions) {
+		r.addLabel("story", story)
+	}
+}
+
+func Feature(feature string) Option {
+	return func(r hasOptions) {
+		r.addLabel("feature", feature)
+	}
+}
+
+func Parameter(name string, value interface{}) Option {
```

Comment:
Can you add a func called Parameters that take as argument a map[string]interface{}, and iterate to call addParameter for each interface?
I personally prefer this approach when we have to pass multiple parameters, so having the 2 possibilities would be great

Reply:
Implemented.